src/main/java/org/spongepowered/api/datapack/DataPackManager.java

```diff
@@ -0,0 +1,44 @@
+/*
+ * This file is part of SpongeAPI, licensed under the MIT License (MIT).
+ *
+ * Copyright (c) SpongePowered <https://www.spongepowered.org>
+ * Copyright (c) contributors
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy
+ * of this software and associated documentation files (the "Software"), to deal
+ * in the Software without restriction, including without limitation the rights
+ * to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+ * copies of the Software, and to permit persons to whom the Software is
+ * furnished to do so, subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included in
+ * all copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+ * THE SOFTWARE.
+ */
+package org.spongepowered.api.datapack;
+
+import org.spongepowered.api.registry.ScopedRegistryHolder;
+
+import java.util.Collection;
+
+/**
+ * An interface for all information about data packs.
+ *
+ * Everything in here is subject to change throughout the
+ * lifetime of the server.
+ */
+public interface DataPackManager extends ScopedRegistryHolder {
```

Comment:
So this I feel is outside the scope of this PR.

There is no need for the `DataPack` registry scope (and frankly it wouldn't change anything with the impl in Sponge of this PR).

That said, there is value in knowing all the loaded packs currently. I would put this into a separate PR where we can discuss what further support in the API for the data pack system would look like.

Reply:
Removed